modules/backup-and-restore/pages/backup-cluster.adoc

```diff
@@ -39,9 +39,15 @@ $ gadmin backup create weekly
 
 This creates a backup file named `weekly-<timestamp>` at the xref:configurations.adoc[backup location you configured].
 
+
 [#_metadata_backup_optional]
 === Metadata backup (Optional)
 
+IMPORTANT: Since v3.9.3, it is no longer required to create a backup for a backups metadata.
+A backup’s metadata is now stored as a part of a backup's files and will be searched for with the backups position.
+
+NOTE: If you are using a version previous to v3.9.3, please refer to the instructions below.
```

Comment:
v3.9.2

Reply:
Thank you. Changed this with latest commit.